packages/devtools_app/test/trees_test.dart

```diff
@@ -274,7 +272,7 @@ void main() {
           var visitedCount = 0;
           breadthFirstTraversal(
             tree,
-            action: (node) => node.setVisitedOrder(++visitedCount),
+            action: (dynamic node) => node.setVisitedOrder(++visitedCount),
```

Comment:
add type TraversalTestTreeNode to breadthFirstTraversal and depthFirstTraversal calls below so that you can remove all the dynamic types

Reply:
done